### ISSUE TITLE
Linux container created on Windows produces CTP1009: An error occurred while attempting to download the Azure Functions CLI toolset

### ISSUE THREAD

**Issue by chriswill** [Azure/azure-functions-docker#315]
Using Visual Studio 2009, I create a bare Function app, with an http trigger.

Once this is complete, I right-click the project and choose Add Docker Support.  I am running Linux docker containers on a Windows machine.

Pressing F5 to debug run the app, the container appears to build successfully.  I can also build it from the command line:
`Successfully built c9824aa43e49
Successfully tagged cloudscopefunctions:dev`

However, starting the app seems to fail with this error:
`error CTP1009: An error occurred while attempting to download the Azure Functions CLI toolset.
1>Done building project "FunctionApp1.csproj" -- FAILED.`

I already have the cli tools on my Windows host.  How would I get them into the container that I'm building?  Wouldn't the image mcr.microsoft.com/azure-functions/dotnet:3.0 already have them?

**Comment by DanielSSilva** [comment]
Hey @chriswill! Did you figure this out? I was doing some tests and I'm able to run thew container successfully, but one of my colleagues can't. From what I've been reading, it might be related to slow network speeds. How did you sort this?

**Comment by chriswill** [comment]
@DanielSSilva I'm no longer working with containers on this project any longer, so I didn't create or record a solution.

**Comment by vacuum-in** [comment]
In my case, the resolution was updating the package Microsoft.VisualStudio.Azure.Containers.Tools.Target.

`Install-Package Microsoft.VisualStudio.Azure.Containers.Tools.Targets -Version 1.15.1`

**Comment by vivekjilla** [comment]
This was resolved in-thread by updating the Microsoft.VisualStudio.Azure.Containers.Tools.Targets package to 1.15.1. Closing as resolved. With no activity since May 21, 2022 (about 4 years), please open a new issue if it recurs on a current supported image.
